azure-pipelines-steps.yml

```
@@ -23,7 +23,7 @@ steps:
   condition: and(succeeded(), eq(variables['Agent.OS'], 'Linux'))
 
 # Install the package
-- script: 'python -m pip install --upgrade pip && pip install --upgrade setuptools && pip install .'
+- script: 'python -m pip install --upgrade pip && pip install --upgrade setuptools && pip install -e .[automl]'
```

I don't think we need the install to be editable.
```suggestion
- script: 'python -m pip install --upgrade pip && pip install --upgrade setuptools && pip install .[automl]'
```